lib/OutlineSegmentCandidatePointSolver/OutlineSegmentCandidatePointSolver.ts

```diff
@@ -372,17 +373,20 @@ export class OutlineSegmentCandidatePointSolver extends BaseSolver {
    */
   private getRotatedComponentPads() {
     const rotationRadians = (this.componentRotationDegrees * Math.PI) / 180
-    return this.componentToPack.pads.map((pad) => ({
-      ...pad,
-      size:
-        (this.componentRotationDegrees + 90) % 180 === 0
-          ? {
-              x: pad.size.y,
-              y: pad.size.x,
-            }
-          : pad.size,
-      offset: rotatePoint(pad.offset, rotationRadians),
-    }))
+    return this.componentToPack.pads.map((pad) => {
+      const normalizedPad = normalizePad(pad)
+      return {
+        ...normalizedPad,
+        size:
+          (this.componentRotationDegrees + 90) % 180 === 0
+            ? {
+                x: normalizedPad.size.y,
+                y: normalizedPad.size.x,
+              }
+            : normalizedPad.size,
+        offset: rotatePoint(normalizedPad.offset, rotationRadians),
+      }
+    })
```

Comment:
is this formatting? If so can we do formatting in a separate PR? Especially with algorithm PRs we want to keep them as small as possible